html/header.php

```diff
@@ -69,20 +70,46 @@
 				width: 100% !important;
 			}
 		</style>
+		<script src="./js/color-toggle.js" defer>
 	</head>
 	<body onUnload="document.getElementById('secret').value = ''">
-		<script>
+	<script>
 			if (location.protocol != 'https:') {
 				document.write('<div style="padding-top: 1%" class="container"><div class="alert alert-danger"><strong>Danger!</strong> This site is not being accessed over an encrypted connection. Do NOT input any sensitive information!</div></div>');
 			}
-			function copyText() {
-				var textToCopy = document.getElementById("copy");
-				textToCopy.select();
-				document.execCommand("copy");
-			}
```

Comment:
Any reason we're adding this same block to multiple pages instead of just once?

Reply:
This was removed and added to the individual pages after adding the theme toggle. After doing some debugging in html/header.php, I discovered a missing closing script tag. All fixed up in [8171400](https://github.com/AndrewPaglusch/FlashPaper/pull/159/commits/8171400912bc3ba6b3135d2e04231c90ee80b1bb)